src/appsignal/cli/install.py

```diff
@@ -23,6 +23,8 @@
 
 INSTALL_FILE_NAME = "__appsignal__.py"
 
+WARNING_EMOJI = "\u26A0\ufe0f"
```

Comment:
Could this be just the plain emoji?

Reply:
I gave that another try but VSCode insists on saving the non-emoji variant (⚠ not ⚠️) to the file. :(